src/main/java/core/basesyntax/maketransaction/IFruitOperation.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.maketransaction;
+
+public interface IFruitOperation {
```

Comment:
```suggestion
public interface FruitOperation {
```
it's not very popular to append `I` letter to mark it as an interface in Java. It's more C# style